keel-sql/src/main/resources/db/changelog/20200731-add-artifact-reference-column.yml

```diff
@@ -0,0 +1,71 @@
+databaseChangeLog:
+  - changeSet:
+      id: add-artifact-type-column
+      author: gyardeni
+      changes:
+        - addColumn:
+            tableName: environment_artifact_queued_approval
+            columns:
+              - name: artifact_reference
+                type: varchar(255)
+                afterColumn: artifact_version
+                constraints:
+                  nullable: true
+        - addColumn:
+            tableName: environment_artifact_constraint
+            columns:
+              - name: artifact_reference
+                type: varchar(255)
+                afterColumn: artifact_version
+                constraints:
+                  nullable: true
+      rollback:
+        - dropColumn:
+            tableName: environment_artifact_constraint
+            columnName: artifact_reference
+        - dropColumn:
+            tableName: environment_artifact_queued_approval
+            columnName: artifact_reference
```

Comment:
I don't think you need this. Most Liquibase migrations have auto-rollback. It's only really when you're doing something unusual (e.g. with `sql`) that you need an explicit rollback.

Reply:
thanks! is this true for the index rollback below as well?